frontend/components/compliance/ListOfCandidateResults.tsx

```diff
@@ -0,0 +1,155 @@
+import { useEffect, useState } from 'react';
+import InfiniteScroll from 'react-infinite-scroll-component';
+import useTranslations from '../../hooks/useTranslation';
+import { getComplianceList } from '../../service/serviceAPI';
+import { CellValue, DataType } from '../../service/types';
+import Table from '../table/Table';
+import InfoModal from '../shared/modals/InfoModal';
+import Button from '../shared/buttons/Button';
+import { COMPLIANCE_TESTING_FORM } from '../../service/constants';
+import InfiniteScrollCustomLoader from '../InfiniteScrollLoader';
+import EvaluationSchemaTable from './EvaluationSchemaTable';
+
+const ListOfCandidateResults = () => {
+  const [resultData, setResultData] = useState<DataType>({ rows: [] });
+  const [currentDataLength, setCurrentDataLength] = useState(0);
+  const [allDataLength, setAllDataLength] = useState(0);
+  const [displayEvaluationSchemaModal, setDisplayEvaluationSchemaModal] =
+    useState(false);
+
+  const { format } = useTranslations();
+
+  const headers = [
+    'table.software_name.label',
+    'table.bb_specification.label',
+    'table.bb_version.label',
+    'test_table.status.label',
+    'table.submission_date.label',
+    'table.deployment_compliance.label',
+    'table.requirement_specification_compliance.label',
+    'table.interface_compliance.label',
+  ];
+
+  const fetchData = async (offset: number, limit: number) => {
+    const data = await getComplianceList(offset, limit);
+    if (data.status) {
+      setAllDataLength(data.data.count);
+
+      const transformedData: DataType = {
+        // @ts-ignore
+        rows: [...resultData.rows],
+      };
+
+      const propertyOrder = [
+        'softwareVersion',
+        'bb',
+        'bbVersion',
+        'status',
+        'submissionDate',
+        'deploymentCompliance',
+        'requirementSpecificationCompliance',
+        'interfaceCompliance',
+      ];
+
+      /*eslint no-prototype-builtins: */
```

Comment:
It would be better to change the code sligthly than to ignore this rule. You can see an example of how to use hasOwnProperty here:
https://eslint.org/docs/latest/rules/no-prototype-builtins